appcheck/firebase-appcheck/src/main/java/com/google/firebase/appcheck/internal/DefaultTokenRefresher.java

```diff
@@ -94,20 +93,12 @@ private long getNextRefreshMillis() {
   }
 
   private void onRefresh() {
-    Task<AppCheckTokenResult> task = firebaseAppCheck.fetchTokenFromProvider();
-    task.addOnCompleteListener(
-        new OnCompleteListener<AppCheckTokenResult>() {
+    Task<AppCheckToken> task = firebaseAppCheck.fetchTokenFromProvider();
```

Comment:
Nit: you probably don't need to declare the variable `task` - you could just call:
firebaseAppCheck.fetchTokenFromProvider().addOnFailureListener(...

Reply:
Done.